pkg/agent/openflow/network_policy.go

```diff
@@ -1450,6 +1450,9 @@ type flowUpdates struct {
 func (c *client) getMatchFlowUpdates(conj *policyRuleConjunction, newPriority uint16) (add, del []binding.Flow) {
 	allClause := []*clause{conj.fromClause, conj.toClause, conj.serviceClause}
 	for _, c := range allClause {
```

Comment:
not introduced by your change; but the var `c` is used for `client` and for the looping var

Reply:
Thanks for catching it, updated.